### PR TITLE
Heirlooms have additional examine text, rather than being renamed

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Heirlooms are no longer named "Cherry family bag of dice", but
rather their heirloom status can be determined by the owner on examine.
/:cl:

Because names are inherent to an object when anyone looks at it, I feel
it can be somewhat confusing that you can tell that this screwdriver
is a precious family heirloom just because of its name, rather than
because of the significance that someone holds it.

Only the owner of the heirloom receives the special examine text; so I'm sure this could probably be used to bluff ownership of something non-standard, if and when heirlooms become more exotic.

Heirlooms still work in the same way though, mood bonus from holding, mood penalty if you don't have it.

### CODE REVIEW

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -146,6 +149,8 @@
 
 /datum/quirk/family_heirloom/on_clone(data)
 	heirloom = data
+	var/datum/component/heirloom/C = heirloom.GetComponent(/datum/component/heirloom)
+	C.owner = quirk_holder
```
Comment (ninjanomnom) [inline code/datums/traits/negative.dm]: You should use the mind instead of the mob so this doesn't need to be updated

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/datums/traits/negative.dm (hunk 1 above)
Comment (coiax) [reply code/datums/traits/negative.dm]: I agree, but quirks are stored on the mob level, and copied across during cloning, because I don't know why

Comment (ninjanomnom) [reply code/datums/traits/negative.dm]: I'm saying this because I don't want a get component call. I don't care how other systems handle it.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-1). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**